DBusInterface.md

```diff
@@ -236,15 +236,14 @@ String of the current special mapping (if mapped to special)
 Array of uints, first entry is the keycode, other entries, if any, are
 modifiers (if mapped to key)
 #### `ActionType`
-- type: `s`, read-only, mutable
-
-String describing the action type of the button ("none", "button", "key",
-"special", "macro", "unknown"). This decides which Mapping  property has a
-value
+- type: `u`, read-only, mutable
 
+An enum describing the action type of the button, see
+ratbag\_button\_get\_action\_type for the list of enums.
+This decides which Mapping  property has a value.
 #### `ActionTypes`
-- type: `as`, read-only, constant
-Array of strings, possible values for ActionType
+- type: `au`, read-only, constant
+Array of enum ratbag\_button\_action\_type, possible values for ActionType
```

Comment:
Might want to amend here that this is for the current device and not just the enum values.